purchases/src/main/kotlin/com/revenuecat/purchases/common/events/BackendStoredEvent.kt

```diff
@@ -73,3 +74,28 @@ internal fun PaywallEvent.toBackendStoredEvent(appUserID: String): BackendStored
         ),
     )
 }
+
+/**
+ * Converts a `PaywallEvent` into a `BackendStoredEvent.Paywalls` instance.
+ *
+ * @receiver The `PaywallEvent` to be converted.
+ * @param appUserID The user ID associated with the event.
+ * @return A `BackendStoredEvent.Paywalls` containing a `BackendEvent.Paywalls`.
+ */
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+internal fun CustomerCenterEvent.toBackendStoredEvent(appUserID: String): BackendStoredEvent {
+    return BackendStoredEvent.CustomerCenter(
+        BackendEvent.CustomerCenter(
+            id = creationData.id.toString(),
+            revisionID = data.revisionID,
+            type = data.type.value,
+            appUserID = appUserID,
+            appSessionID = data.sessionIdentifier.toString(),
```

Comment:
For context, `appSessionID` differs from Paywall's `sessionID` in that `appSessionID` is per app session. This is something we added for Customer Center and is an id that gets generated whenever the SDK is configured. The regular Session ID that Paywalls uses is used to track a paywalls session.

Reply:
I'll update the docs